std_msgs/Header header
string imageTopic
Rectangle[] rect
string globalImageLabel
